src/main/java/com/yahoo/bullet/rest/controller/HTTPQueryController.java

```diff
@@ -43,13 +45,16 @@
         String queryID = QueryService.getNewQueryID();
         try {
             query = preprocessingService.convertIfBQL(query);
+            preprocessingService.throwIfQueryLimitReached(queryService);
```

Comment:
make it consistent with containsWIndow?   both throw exceptions or both return boolean?